app/views/candidate_interface/start_page/show.html.erb

```diff
@@ -32,13 +32,21 @@
       You can teach in further education (age 14 to adult), or study for a degree leading to qualified teacher status.
       <%= govuk_link_to('Learn more about eligibility for teacher training', 'https://getintoteaching.education.gov.uk/eligibility-for-teacher-training', target: :_blank) %>.
     </p>
-    <%= link_to candidate_interface_eligibility_path, role: 'button', class: 'govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-4 govuk-button--start', 'data-module': 'govuk-button' do %>
-      <%= t('application_form.begin_button') %>
-      <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false">
-        <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
-      </svg>
+
+    <% if FeatureFlag.active?('create_account_or_sign_in_page') %>
+      <%= link_to candidate_interface_account_prompt_path, role: 'button', class: 'govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-4 govuk-button--start', 'data-module': 'govuk-button' do %>
+        <%= t('application_form.begin_button') %>
+        <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false">
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+        </svg>
+      <% end %>
+    <% else %>
+      <%= link_to candidate_interface_eligibility_path, role: 'button', class: 'govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-4 govuk-button--start', 'data-module': 'govuk-button' do %>
+        <%= t('application_form.begin_button') %>
+        <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false">
+          <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+        </svg>
+      <% end %>
     <% end %>
-    <p class="govuk-body">or</p>
-    <p class="govuk-body"><%= govuk_link_to 'Sign in to an existing application', candidate_interface_sign_in_path %></p>
```

Comment:
I'm not sure we should be removing this completely. I think this needs to be under the `create_account_or_sign_in_page` feature flag.